src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTArgumentList.cs

```diff
@@ -2989,26 +2989,19 @@ public void AddExtObject24()
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.Stream, DocType.XPathDocument)]
         [InlineData(InputType.Navigator, ReaderType.XmlValidatingReader, TransformType.TextWriter, DocType.XPathDocument)]
         [Theory]
-        public void AddExtObject25(InputType inputType, ReaderType readerType, TransformType transformType, DocType docType)
+        public void TC_ExtensionObj_Function_Mismatch_IncorrectCasing(InputType xslInputType, ReaderType readerType, TransformType outputType, DocType navType)
         {
             MyObject obj = new MyObject(25, _output);
             m_xsltArg = new XsltArgumentList();
 
             m_xsltArg.AddExtensionObject(szDefaultNS, obj);
-            if (LoadXSL("MyObject_CaseSensitive.xsl", inputType, readerType) == 1)
+            LoadXSL("MyObject_CaseSensitive.xsl", xslInputType, readerType);
```

Comment:
It throws an exception or always returns 1. The method can be changed to void but I don't want to do that now to not increase the scope of the PR too much.